### PR TITLE
Add API to enable non-3D controls

### LINKED ISSUE #5773: How to add Native UI : Windows.UI.Xaml.Controls.InkCanvas + Unexpected Behavior?
I have been unable to get the drawing strokes to register/render on the [InkCanvas ](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.inkcanvas?view=winrt-19041)as a Native UI component. 
 
I've created the view manager as so:
InkCanvasViewManager.cs
```c#
using Microsoft.ReactNative;
using Microsoft.ReactNative.Managed;
using Windows.UI;
using Windows.UI.Input.Inking;
using Windows.UI.Xaml;
using Windows.UI.Xaml.Controls;

namespace ReactNativeInk
{
    class InkCanvasViewManager : IViewManager
    {
        private InkCanvas inkCanvas;

        public string Name => "InkCanvasCS";

        public FrameworkElement CreateView()
        {
            inkCanvas = new InkCanvas();

            inkCanvas.InkPresenter.InputDeviceTypes = Windows.UI.Core.CoreInputDeviceTypes.Mouse | Windows.UI.Core.CoreInputDeviceTypes.Pen | Windows.UI.Core.CoreInputDeviceTypes.Touch;

            return inkCanvas;

        }
    }
}
```

I have gotten other components to work and render fine (Slider, InkToolBar, CalendarView), but just not this particular canvas. 

Out of ideas for where the source of the problem could be. One caveat of the InkCanvas is that it requires ` inkCanvas.InkPresenter.InputDeviceTypes` to be set in other for the ink strokes to render, is this somehow being undone after CreateView()?

Any help/pointers would be greatly appreciated.
Thank you!

### PR DESCRIPTION
**Pull request by asklar**
Some controls like `InkCanvas` do not work in environments where a 3D transform is present. RNW adds a `PerspectiveTransform3D` at the root in order to achieve a projective perspective look during rotations (the default for XAML is orthographic). This PR adds an API to turn off the perspective transform which enables controls like `InkCanvas` to work and receive input.

Fixes #5773 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/5798)

### CODE REVIEW

File: vnext/Microsoft.ReactNative/ReactRootView.h [hunk 1]
```diff
@@ -35,8 +44,10 @@ struct ReactRootView : ReactRootViewT<ReactRootView> {
   hstring m_componentName;
   ReactNative::JSValueArgWriter m_initialPropsWriter;
   folly::dynamic m_initialProps;
-
+  bool m_isPerspectiveEnabled;
```
Comment (kmelmon) [inline vnext/Microsoft.ReactNative/ReactRootView.h:47]: >m_isPerspectiveEnabled [](start = 7, length = 22)

Flag is not initialized.

**Comment by kmelmon** [comment]
This change needs some thought...

We need to decide what the default should be.  On the other platforms you don't have to do anything to get perspective so that would argue for making the default true.  However this then causes issues with several scenarios out of the box (InkCanvas and Popup bug #4466) and customers likely won't know why.  Having true 3D transforms are somewhat rare, and putting a 3D transform on the root of the tree is a fairly expensive thing that is having unwanted side-effects so that argues for making false the default.

Note also I still have bug #4466 assigned to me to fix which is closely related.  It's possible this could be fixed by applying perspective only to the elements that actually have a 3D transform, and that could make this flag unnecessary, but there are a number of edge cases and transition cases to work through, so that makes it tricky, so I can't guarantee that will be the final solution.

Overall I'm not convinced having perspective on the root is the right long term solution.  And you could argue nobody should have to think about what to set it to - it should "just work".  Having said all that, as a short term solution, this might be fine.  If I do manage to find a way to add perspective on demand rather than on the root I can remove this and deal with the breaking change, or find a way to respect the flag.

File: vnext/Microsoft.ReactNative/ReactRootView.h (hunk 1 above)
Comment (asklar) [reply vnext/Microsoft.ReactNative/ReactRootView.h:47]:
```suggestion
  bool m_isPerspectiveEnabled{false};
```

**Comment by acoates-ms** [comment]
The root view interface in question already inherits from a XAML object, so that part of this should be fine.

This property is obviously going to need some documentation.

One thing that facebook does when adding APIs that it doesn't know if there will be long term support for is that they call them:  unstable_*.   This is sort of like putting them in the experimental namespace in windows. -- Just a thought.

**Comment by NickGerleman** [comment]
> The root view interface in question already inherits from a XAML object, so that part of this should be fine.

Oh yeah, you're totally right. I get the feeling that will need to get refactored eventually, but good to know it isn't clean yet. Agree with sentiments of setting expectations around API stability if we can/making this a bit more hidden. Like the ideas of adding "experimental" or whatever to the property name, or moving the flag to a less prominent place.

**Pull request merged**